staging/cert-manager-setup/templates/clusterissuer.yaml

```diff
@@ -1,6 +1,6 @@
 # DEPRECATED, this file should be deleted soon
```

Comment:
Shouldn't we now finally delete this file?

Reply:
:+1: